src/handlers/base.ts

```diff
@@ -6,6 +8,8 @@ import {
 } from '../userTypes'
 import { InputError } from './types'
 
+dotenv.config()
+
```

Comment:
I don't think we should load the `.env` file ourselves. It's not our responsibility to load the environment variables. The user is expected to do that however they wish which may or may not involve using a .env file. For context, basically no one uses `.env` files to load variables in production they're always injected directly into the environment so I don't think it makes sense to be this opinionated about it.

I would have a different opinion on this if we were a CLI intended to be used from the command line in a project, but since this is a package intended to be used in production we should not do this.

Also `dotenv` is installed as a dev dependency which means this will fail if we tried to do it in production.